test/unit/ngsiv2/httpBindings-test.js

```diff
@@ -0,0 +1,738 @@
+/*
+ * Copyright 2016 Telefonica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of iotagent-ul
+ *
+ * iotagent-ul is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * iotagent-ul is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with iotagent-ul.
+ * If not, seehttp://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License
+ * please contact with::[iot_support@tid.es]
+ *
+ * Modified by: Fernando Méndez, Daniel Calvo - ATOS Research & Innovation
+ */
+
+ 'use strict';
+
+var iotagentUl = require('../../../'),
+config = require('./config-test.js'),
+nock = require('nock'),
+iotAgentLib = require('iotagent-node-lib'),
+should = require('should'),
+async = require('async'),
+request = require('request'),
+utils = require('../../utils'),
+contextBrokerMock,
+iotamMock;
+
+ describe('HTTP Transport binding: measures', function() {
+    beforeEach(function(done) {
+        var provisionOptions = {
+            url: 'http://localhost:' + config.iota.server.port + '/iot/devices',
+            method: 'POST',
+            json: utils.readExampleFile('./test/unit/ngsiv2/deviceProvisioning/provisionDevice1.json'),
+            headers: {
+                'fiware-service': 'smartGondor',
+                'fiware-servicepath': '/gardens'
+            }
+        };
+
+        nock.cleanAll();
+
+        iotamMock = nock('http://localhost:8082')
```

Comment:
Doesn't follow the usual indent style used in other mock definitions in other files in this same PR.

(Have a general look to this in the file... it seems it happends more times, although I'm not going to comment again in each point, to not add diff-noise to the PR)